requirements_test.txt

```diff
@@ -6,6 +6,7 @@
 # contain hints for available should be kept in sync with them
 
 -c homeassistant/package_constraints.txt
+-r requirements.txt
```

Comment:
Including this here makes the purpose of the changed file unclear.

Maybe instead update the task to install from `requirements.txt` too?

Reply:
Alternatively it would make sense to include these requirements in `requirements_test_all.txt` since that file purpose, in my opinion, is to install all requirements to run tests.